feature-libs/checkout/components/components/payment-method/payment-method.component.ts

```diff
@@ -158,6 +158,12 @@ export class PaymentMethodComponent implements OnInit, OnDestroy {
   }
 
   selectPaymentMethod(paymentDetails: PaymentDetails): void {
+    this.globalMessageService.add(
+      {
+        key: 'paymentMethods.paymentMethodSelectedSuccess',
```

Comment:
let's remove success from the key since we're using it to display as 'info' to have it vocalized this way instead.